components/layout/SideBar.tsx

```
@@ -0,0 +1,145 @@
+            );
+          })}
+        </nav>
+
+        {/* ─── Bottom section: collapse toggle ─── */}
+        <div className="mt-auto flex shrink-0 flex-col gap-2 px-3 pb-6">
+          {/* Collapse/Expand toggle (desktop only) */}
+          <button
+            onClick={() => setIsCollapsed(!isCollapsed)}
+            className="hidden items-center justify-center gap-3 rounded-xl px-4 py-3 text-white/40 transition-all duration-200 hover:bg-white/5 hover:text-white lg:flex"
+            aria-label={isCollapsed ? "Expand sidebar" : "Collapse sidebar"}
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)